includes/admin/settings/register-settings.php

```diff
@@ -886,7 +886,15 @@ function edd_get_registered_settings() {
 					'show_agree_to_terms' => array(
 						'id'   => 'show_agree_to_terms',
 						'name' => __( 'Agree to Terms', 'easy-digital-downloads' ),
-						'desc' => __( 'Check this to show an agree to terms on checkout that users must agree to before purchasing.', 'easy-digital-downloads' ),
+						'desc' =>
+								__( 'Check this to show an agree to terms on checkout that users must agree to before purchasing.', 'easy-digital-downloads' ) .
+								'<p>' .
+								sprintf(
+										__( 'Need help creating a Terms of Agreement? We recommend using %sTermageddon%s.', 'easy-digital-downloads' ),
```

Comment:
Is this the language they requested? `a Terms of Agreement` sounds so weird to me.

Reply:
@ashleyfae Yes that is the copy they supplied to us.